android-client-sdk/src/main/java/com/devcycle/sdk/android/eventsource/DVCLifecycleCallbacks.kt

```diff
@@ -0,0 +1,49 @@
+package com.devcycle.sdk.android.eventsource
```

Comment:
I think this should go in a different directory since it doesnt really have anything to do with event source

Reply:
true, maybe just in the com.devcycle.sdk.android.api directory?

Reply:
that would be better. I also don't like that directory's organization either and we probably need to shuffle some stuff around but that's probably the best place right now.

Reply:
done!